src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -604,3 +537,32 @@ def get_pg_info(cls):
             'statuses': pg_summary['all'],
             'pgs_per_osd': pgs_per_osd,
         }
+
+
+def _generate_key(encryption_type, backend, config_name):
+    if encryption_type == EncryptionTypes.KMS.value:
+        return f'rgw_crypt_{backend}_{config_name}'
+    elif encryption_type == EncryptionTypes.S3.value:
+        return f'rgw_crypt_sse_s3_{backend}_{config_name}'
+
+def _get_conf_keys(fields: List[str], req_fields: List[str], encryption_type: str, backend: str, daemon_name: str) -> Dict[str, str]:
+    config = {}
+    for field in fields:
+        key = _generate_key(encryption_type, backend, field)
+        try:
+            value = CephService.send_command('mon', 'config get', who=name_to_config_section(daemon_name), key=key)
+            if field in req_fields and not (isinstance(value, str) and value.strip()):
+                config = {}
+                break
+
+            config[field] = value.strip() if isinstance(value, str) else value
+        except Exception as e:
+            logger.error("Error fetching Vault config for key %s: %s", key, str(e))
```

Comment:
let's not log this error because this is by intention. Some fields are not there in the config key section which is ok. We don't want to log the whole thing as an error whenever which misleads the user whenever he vists the encryption page.
```suggestion
            pass
```

Reply:
fixed